core/controllers/collection_editor.py

```diff
@@ -103,15 +103,8 @@ def get(self, collection_id):
             collection_id, strict=False)
 
         self.values.update({
-            'can_edit': True,
-            'can_unpublish': rights_manager.Actor(
-                self.user_id).can_unpublish(
-                    feconf.ACTIVITY_TYPE_COLLECTION, collection_id),
             'collection_id': collection.id,
-            'is_private': rights_manager.is_collection_private(collection_id),
             'nav_mode': feconf.NAV_MODE_CREATE,
-            'owner_names': rights_manager.get_collection_owner_names(
-                collection_id),
             'title': collection.title,
```

Comment:
Nope, because we're handling this via the `get` in the AJAX call, we no longer need this to be here.